### PR TITLE
docs(readme): add badges

### PR DESCRIPTION
**Pull request by philibea**
Add usefull badge on README

### CODE REVIEW

File: static/images/scaleway-logo.svg [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (chambo-e) [inline static/images/scaleway-logo.svg:1]: You don't need to inline images now, GitHub allow to update them directly on their servers 🙂

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+![Scaleway Logo](./static/images/scaleway-logo.svg)
+
+
+![Codecov](https://img.shields.io/codecov/c/github/scaleway/scaleway-lib)
+![GitHub closed issues](https://img.shields.io/github/issues-closed/scaleway/scaleway-lib)
+![dependencies](https://david-dm.org/scaleway/scaleway-lib.svg)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)
```
Comment (chambo-e) [inline README.md:7]:
```suggestion
[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)
```

Kinda useless this one non ?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+![Scaleway Logo](./static/images/scaleway-logo.svg)
+
+
+![Codecov](https://img.shields.io/codecov/c/github/scaleway/scaleway-lib)
+![GitHub closed issues](https://img.shields.io/github/issues-closed/scaleway/scaleway-lib)
+![dependencies](https://david-dm.org/scaleway/scaleway-lib.svg)
```
Comment (chambo-e) [inline README.md:6]: This one doesn't work :)

**Review by chambo-e — CHANGES_REQUESTED** [review]
Screenshot 2021-03-15 at 10 48 21

The Scaleway logo is way too big no ?

File: README.md (hunk 3 above)
Comment (philibea) [reply README.md:6]: It's work fine for me 
Capture d’écran 2021-03-16 à 10 35 10

**Pull request closed without merging**